Project failing to build

When running `npm install` from the root, I end up with [the following results](https://gist.github.com/astorm/b77d88f29c55f5fb7717e87af0df5b76).  TLDR -- is _seems_ like the prometheus package at `master` can't compile right now. 

```
    test/prometheus.test.ts:527:5 - error TS7027: Unreachable code detected.

    527     done();
            ~~~~~~~


    Found 2 errors.

    npm ERR! code ELIFECYCLE
    npm ERR! errno 1
    npm ERR! @opentelemetry/exporter-prometheus@0.7.0 compile: `npm run version:update && tsc -p .`
    npm ERR! Exit status 1
    npm ERR! 
    npm ERR! Failed at the @opentelemetry/exporter-prometheus@0.7.0 compile script.
    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

    npm ERR! A complete log of this run can be found in:
    npm ERR!     /Users/alanstorm/.npm/_logs/2020-05-07T03_59_24_975Z-debug.log
    lerna info lifecycle @opentelemetry/exporter-collector@0.7.0~prepare: @opentelemetry/exporter-collector@0.7.0
    lerna info lifecycle @opentelemetry/exporter-jaeger@0.7.0~prepare: @opentelemetry/exporter-jaeger@0.7.0
    lerna info lifecycle @opentelemetry/exporter-zipkin@0.7.0~prepare: @opentelemetry/exporter-zipkin@0.7.0
    lerna info lifecycle @opentelemetry/node@0.7.0~prepare: @opentelemetry/node@0.7.0
    lerna info lifecycle @opentelemetry/shim-opentracing@0.7.0~prepare: @opentelemetry/shim-opentracing@0.7.0
    lerna info lifecycle @opentelemetry/test-utils@0.7.0~prepare: @opentelemetry/test-utils@0.7.0
    lerna info lifecycle @opentelemetry/web@0.7.0~prepare: @opentelemetry/web@0.7.0
    lerna info lifecycle @opentelemetry/exporter-prometheus@0.7.0~prepare: Failed to exec prepare script.  
```

_Originally posted by @astorm in https://github.com/open-telemetry/opentelemetry-js/pull/1032#issuecomment-625028942_

--- a/packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts
+++ b/packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts
@@ -522,8 +522,5 @@ describe('PrometheusExporter', () => {
 });
 
 function errorHandler(done: Mocha.Done): (err: Error) => void {
-  return () => {
-    assert.ok(false, 'error getting metrics');
-    done();
-  };
+  return (err) => done(err);
 }

--- a/packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts
+++ b/packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts
@@ -522,5 +522,5 @@ describe('PrometheusExporter', () => {
 });
 
 function errorHandler(done: Mocha.Done): (err: Error) => void {
-  return (err) => done(err);
+  return err => done(err);
 }

